common/ripple/scoreUtils.py

```diff
@@ -14,8 +18,15 @@ async def overwritePreviousScore(
     The ratelimit has already been checked in the case of the !overwrite command.
     """
 
+    # XXX: a bit of a strange dependency -- means the user needs to be online
```

Comment:
this is... not new.... how are they going to send `!overwrite` if they aren't online?

Reply:
It's new in the context of this function, but yeah the only existing invocation is from the command